game/languages/French.ini

```diff
@@ -893,7 +893,6 @@ ABOUT_MAIN=À propos...
 ABOUT_MAIN_DESC=Ultrastar Deluxe
 ABOUT_MAIN_WHEREAMI=À propos...
 ABOUT_USDX_CREDITS=Credits
-;TODO: ABOUT_OVERVIEW=UltraStar Deluxe is a free open-source karaoke game for your PC. The gameplay experience is similar to that of the commercial product SingStar™ by Sony Computer Entertainment, which is exclusively available for the Playstation®. Unlike SingStar™, however, UltraStar Deluxe allows users to create their own songs and sing them on their PC.\nBy publishing the sourcecode, we’ve enabled you to customize the game completly to your heart’s content, desires and particular needs.\nEspecially you are allowed to modify the product as you wish.\n\nCreate your own themes or download content from the community, such as plugins, free songs or code patches for extended features.\nIn party mode it allows up to twelve players to sing along to your favorite music in order to score points, depending on the pitch of the voice and the rhythm of singing.
-
+ABOUT_OVERVIEW=UltraStar Deluxe est un jeu de Karaoké open-sources gratuit pour votre ordinateur. L'expérience de jeu est similaire au produit commercial SingStar™ de Sony Computer Entertainment, qui n'est disponible que sur les plateformes Playstation®. Contrairement à SingStar™, UltraStar Deluxe vous permet de créer et chanter vos propres chansons. \nEn publiant le code source, nous vous permettons de personnaliser complètement le jeu comme il vous le souhaitez ou pour des besoins particuliers.\nVous êtes spécialement autorisé à modifier le produit comme bon vous semble. \n\nCréer vos propres thèmes ou télécharger du contenu depuis la communauté, tels que des plugins, chansons gratuites ou de nouvelles fonctionnalités.\nDans le mode party, vous pouvez être jusqu'à douze joueurs pour chanter votre musique favorite dans le but de gagner des points en fonction de la hauteur et du rythme de la voix chantante. 
```

Comment:
I pasted this comment over the already committed one, but it did not lead to any changes. When resolving stuff, please pay special attention to this one. If there's meant to be a special space somewhere, I can't copy those from github comments apparently.
I'm not sure where the change should be, since it's a wall of text and French is not one of my languages.